src/lib/src/services/notifier-timer.service.ts

```diff
@@ -22,7 +22,7 @@ export class NotifierTimerService {
 	/**
 	 * Timeout ID, used for clearing the timeout later on
 	 */
-	private timerId: number;
+	private timer: NodeJS.Timer;
```

Comment:
I think this is an issue introduced by the NodeJS typings, the timer ID here should be of type `number` (as before), not `NodeJS.Timer`. Perhaps we should cast this into a number, if necessary?

Reply:
You are right, TypeScript was complaining about this, I didn't check the signature of the method setTimeout which is wrong https://github.com/Microsoft/TypeScript/issues/842 I'll use window.setTimeout instead.